arch/riscv/Kconfig.core

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 2022 Carlo Caione <ccaione@baylibre.com>
+# SPDX-License-Identifier: Apache-2.0
+
+menu "RISCV core"
+
+config RISCV_CORE_E31
+	bool "Use E31 core"
+	select RISCV_PMP
+	select ARCH_HAS_USERSPACE
+	select ARCH_HAS_STACK_PROTECTION if PMP_STACK_GUARD
+	select RISCV_ISA_EXT_M
+	select RISCV_ISA_EXT_A
+	select RISCV_ISA_EXT_C
+	default n
```

Comment:
`default n` should not be necessary here.

Reply:
ok